Burrows_Wheeler_Transformation_Algorithm/BWT.py

```diff
@@ -0,0 +1,25 @@
+def bwt(s):
+    assert "\002" not in s and "\003" not in s, "Input string cannot contain STX and ETX characters"
+    s = "\002" + s + "\003"
+    table = sorted(s[i:] + s[:i] for i in range(len(s)))
+    last_column = [row[-1:] for row in table]
+    return "".join(last_column)
+
+
+#----INVERSE BURROWS WHEELER TRANSFORMATION
+#This is to get back the original message out of the compressed version.
+def ibwt(r):
```

Comment:
What is `ibwt`?

Reply:
@jainaman224, ibwt or Inverse Burrows-Wheeler Transformation is to compute the inverse of BWT i.e. to undo the effects of BWT and recover the original string.

Reply:
And I closed the previous PR #690 also.